.changeset/slow-pugs-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade-mcp": minor
+---
+
+feat(blade-mcp): add changelog tool
```

Comment:
minor will work but honestly this is a major upgrade

Reply:
But in any case, general breaking changes and semver may not have big impact in MCPs because they just get updated and toolcalls are not a concern of user but rather MCP client.